liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/generateChangelog.test.groovy

```diff
@@ -13,12 +12,17 @@ CommandTests.define {
 Short Description: Generate a changelog
 Long Description: Writes Change Log XML to copy the current state of the database to standard out or a file
 Required Args:
-  changelogFile (String) File to write changelog to
   url (String) The JDBC database connection URL
```

Comment:
@filipelautert, hello! I see we don't have a test for the case where `generate-changelog` is executed without providing the required `url` argument. What do you think about adding one?

Reply:
I think it would also be good to add a test "Running generateChangeLog should add changesets in the correct order". I saw one similar to that in the diffChangelog.test.groovy.

Reply:
Hello! Sure, tests added.